filebeat/inputsource/udp/server.go

```diff
@@ -71,3 +72,10 @@ func (u *Server) createConn() (net.PacketConn, error) {
 
 	return listener, err
 }
+
+func (u *Server) network() string {
+	if u.config.Network != "" {
+		return u.config.Network
+	}
+	return "udp"
```

Comment:
[Suggestion]
Same thing here:
```suggestion
	return networkUDP
```